runbotics/runbotics-desktop/src/action/mail/mail.action-handler.ts

```diff
@@ -73,11 +74,15 @@ export default class MailActionHandler extends StatelessActionHandler {
     }
 
     run(request: MailActionRequest) {
-        const mailCredential: MailCredential = { // @todo replace with method for matching credentialId/templateName with decrypted credentials
+        const passwordManagerCredential =
+            credentialAttributesMapper<MailCredential>(request.credentials);
+
+        // @todo After completion of password manager switch fully to passwordManagerCredential
+        const mailCredential: MailCredential = passwordManagerCredential ?? {
             mailUsername: this.serverConfigService.mailUsername,
             mailPassword: this.serverConfigService.mailPassword,
             mailHost: this.serverConfigService.mailHost,
-            mailPort: Number(this.serverConfigService.mailPort),
+            mailPort: this.serverConfigService.mailPort,
```

Comment:
I'm afraid this is not going to work - env values are string, and for mailPort number is expected